app/submitter/convert_payload_0_0_1.py

```diff
@@ -52,7 +61,7 @@ def convert_answers_to_payload_0_0_1(
                     if answer["id"] == answer_in_block.answer_id:
                         answer_schema = answer
 
-                value = answer_in_block.value
+                value: AnswerValueTypes = answer_in_block.value
```

Comment:
Not sure this annotation is needed as it should already know that if it knows it is an answer?.

Reply:
Removed, and it was unnecessary indeed, thanks!